appcoins/src/main/java/cm/aptoide/pt/donations/Donation.kt

```diff
@@ -0,0 +1,4 @@
+package cm.aptoide.pt.donations
+
+
+class Donation(val domain: String, val owner: String, val appc: Float) 
```

Comment:
i dont think we should be receiving null donations...  specially since all this info is mandatory. The only that used to be able to be null is the owner, but as we discussed yesterday with the wallet team, that is not accepted anymore and that also used to be converted server side to Misterious Donor.